teuthology/orchestra/opsys.py

```diff
@@ -73,7 +73,7 @@
 DEFAULT_OS_VERSION = dict(
     ubuntu="22.04",
     fedora="25",
-    centos="8.stream",
+    centos="9.stream",
```

Comment:
in testing i saw that the centos 9 container check wasn't actually happening until i changed this too